### ISSUE TITLE
Traveler's Cloak (Invisibility) and Invisibility potion glitch

### ISSUE THREAD

**Issue by mobslay** [TheIllusiveC4/ConstructsArmory#198]
Versions (Be specific, do not write "latest"):
Construct's Armory: conarm-1.12.2-1.2.3
Tinkers' Construct:1.12.2-2.12.0.135
Mantle: 1.12-1.3.3.49
Forge: 2816
Observed Behavior: When I turn on invisibility for my traveler's cloak or I drink  an invisibility potion all of my armor automatically unequips. I am unable to put the traveler's cloak back on.
Expected Behavior: I am supposed to become invisible and still have my armor equiped
Steps to Reproduce: Get traveler's cloak (invisibility) and put in on a chestplate for construct's armory. Activate the invisibility. (survival mode)
OR
Drink and invis potion with armor on. (survival mode)
...

Crash Log: Please use a paste site like hastebin.

**Comment by TheIllusiveC4** [comment]
I can't reproduce this issue, invisibility works fine for me.
Have you tried reproducing this with only Tinkers' and Construct's Armory installed?
Can you update your mod versions and then try again? It seems like you're a few versions behind.

**Comment by starman2995** [comment]
I had the same problem, and upon some research, I discovered the cause and the solution.
The cause of the problem is most likely that in your modpack, you have the dungeon tactics mod. That mod has a feature automatically enabled that removes all of your armor if you have invisibility. It makes sense if you just need to go invisible, but not if it is your armor granting the buff.
The solution is to go into the file settings for dungeon tactics and disable that specific setting. This website is the one I used to find the issue and also details how to fix it:
https://www.reddit.com/r/feedthebeast/comments/dzqrgy/rad_auto_unequips_my_armor_when_invisible/
